### ISSUE TITLE
Error when passing fractions for gutter sizes

### ISSUE THREAD

**Issue by lmartins** [ericam/susy#408]
With the following settings:

```
$susy: (
    columns: 12,
    gutters: 1/4,
    math: fluid,
    output: float,
    gutter-position: inside,
);
```

I get:

```
17:02:18] gulp-notify: [Error running Gulp] stdin:69: may only compare numbers

Error: { [Error: stdin:69: may only compare numbers
]
  message: 'stdin:69: may only compare numbers\n',
  showStack: false,
  showProperties: true,
  plugin: 'gulp-sass',
  __safety: { toString: [Function] } }
```

If I change that property to a pixel value it compiles without errors. 

The `columns`property` doesn't work when using asymmetrical grids `columns: (1 1 2 3 5 8 13),`.
This returns the following:
```
[gulp] Starting 'sass'...
[gulp] Finished 'sass' after 2.01 ms
[17:09:46] gulp-notify: [Error running Gulp] /Users/luismartins/Sites/LAB/libsass/src/vendor/susy/sass/susy/su/grid:75: index out of bounds for `nth($list, $n)`

Error: { [Error: /Users/luismartins/Sites/LAB/libsass/src/vendor/susy/sass/susy/su/grid:75: index out of bounds for `nth($list, $n)`
]
  message: '/Users/luismartins/Sites/LAB/libsass/src/vendor/susy/sass/susy/su/grid:75: index out of bounds for `nth($list, $n)`\n',
  showStack: false,
  showProperties: true,
  plugin: 'gulp-sass',
  __safety: { toString: [Function] } }
```

Im running `node-sass-1.2.3`

**Comment by reintroducing** [comment]
I'm using gulp-sass 1.3.2 and the latest version of susy.  I'm still getting this error even though when you track this to the libsass ticket as well as the one linked there this should be fixed. Any ideas?  This is all I have in my sass file and it's throwing the same error still.

```
$susy: (
    columns: 12,
    gutters: 1/4
);
```

**Comment by ericam** [comment]
Following the trail of dependencies, I'm not sure that the latest gulp-sass includes the latest libsass yet. Looks to me like node-sass is several versions behind.

- `gulp-sass 1.3.1` (latest) uses `node-sass 2.0.0-beta` (https://github.com/dlmanning/gulp-sass/blob/24cc36a033b0508a9e6ed7628c8153d349759703/package.json)
- `node-sass 2.0.0-beta` uses `libsass 3.1.0-beta` (https://github.com/sass/node-sass/blob/7f17da354852ccb9575a8f5ebeb9995789f29d9f/package.json)

The last two releases of libsass (`3.1.0-beta.2` and `3.1.0`) are not yet available in node-sass, and therefor also not in `gulp-sass`

**Comment by reintroducing** [comment]
Ah, thanks for following up so quickly on this.  That definitely makes sense.  I REALLY want to adopt libsass already due to the incredibly slow compile times from the ruby version but it's still a little rough around the edges :\

**Comment by jamiehutber** [comment]
Just giving everybody a heads up who might be having problems like this.

I had to update libsass to `3.1.0-beta.2` and then inside my sass use `(1/4)` rather than just `1/4`. This will force it to be treated as a number.

Taken from here:
https://github.com/sass/libsass/issues/590

I am unsure if updating `node-sass`'s dependances to use the beta helped at all. But its working right now so I can't test if that was a necessary step.

**Comment by timmyomahony** [comment]
I'm also suddenly having an issue with this. Everything was working fine and somewhere along the chain of dependencies there was obviously and  update that's caused this to show its head. 

I'm using:

    grunt-sass v0.18.0
    node-sass v2.0.1
    libsass (OSX) v3.1.0

I haven't been able to fix it yet.

(I know this isn't an issue with susy but that's how it manifested itself and might be useful for someone searching via Google.)

**Comment by tomazzaman** [comment]
A workaround is to define the gutter width outside map, like so:

```
$gutters: 1/4;

$susy: (
  global-box-sizing: border-box,
  math: static,
  columns: 12,
  column-width: 70px,
  gutters: $gutters,
  container: auto,
  container-position: center,
);
```

**Comment by christianmagill** [comment]
Wrapping fraction in parenthesis also seems to work.

**Comment by ericam** [comment]
It looks to me like `node-sass`, `grunt-sass`, and `gulp-sass` have all been updated — so this should not cause an error any more. Have you tried updating your dependencies?

**Comment by DenisMir** [comment]
@ericam 
I'm still getting this error. 

These are my grid settings: 
```
$grid: (columns: 12, column-width: false, gutters: 1 / 1.618, gutter-position: after, math: fluid);
```

When having something like that: 
```
.col__b0-full {
  @include span(full);
}
```

I'm getting the `may only compare numbers` error. I'm using the latest _node-sass_ version which uses _3.1_.

Any suggestions?

**Comment by Reklino** [comment]
This still seems to be an issue. Here are my grid settings:
```
$susy: (
    columns: 3,
    gutters: 1/4,
    gutter-position: inside,
    container: 450px
);
```
I've started using `.25` instead of `1/4` for now.

**Comment by ericam** [comment]
Yeah, they fixed the error with fractions more generally, but not fractions *retrieved from a map* as far as I can tell. I filed sass/libsass#1159 to track it, and released `2.2.3` with the default changed. I'll keep this issue open, with a blocked-by-sass tag.
